Backend/controllers/reservation.js

```diff
@@ -212,6 +214,15 @@ exports.deleteReservation = async (req, res, next) => {
       }
     );
 
+    if (req.user.role === "admin" && req.user.id !== reservation.user.toString()) {
```

Comment:
In my opinion, moderators should have the ability to force-cancel reservations cuz they're one of the biggest roles

Reply:
thx for the comment kub fix in next commit